bindings/node.js/test.ts

```diff
@@ -46,99 +79,232 @@ describe("C-KZG", () => {
+    it("invalid proof", () => {
+      const commitment = new Uint8Array(BYTES_PER_COMMITMENT).fill(0);
+      commitment[0] = 0xc0;
+      const z = new Uint8Array(BYTES_PER_FIELD_ELEMENT).fill(1);
+      const y = new Uint8Array(BYTES_PER_FIELD_ELEMENT).fill(1);
+      const proof = new Uint8Array(BYTES_PER_PROOF).fill(0);
+      proof[0] = 0xc0;
 
-    expect(verifyKzgProof(commitment, z, y, proof)).toBe(false);
+      expect(verifyKzgProof(commitment, z, y, proof)).toBe(false);
+    });
   });
 
-  it("computes the aggregate proof when for a single blob", () => {
-    let blobs = new Array(1).fill(0).map(generateRandomBlob);
-    let commitments = blobs.map(blobToKzgCommitment);
-    let proof = computeAggregateKzgProof(blobs);
-    expect(verifyAggregateKzgProof(blobs, commitments, proof)).toBe(true);
-  });
+  describe("verifyBlobKzgProof", () => {
+    it("correct blob/commitment/proof", () => {
+      let blob = generateRandomBlob();
+      let commitment = blobToKzgCommitment(blob);
+      let proof = computeBlobKzgProof(blob);
+      expect(verifyBlobKzgProof(blob, commitment, proof)).toBe(true);
+    });
 
-  it("fails when given incorrect commitments", () => {
-    const blobs = new Array(2).fill(0).map(generateRandomBlob);
-    const commitments = blobs.map(blobToKzgCommitment);
-    commitments[0][0] = commitments[0][0] === 0 ? 1 : 0; // Mutate the commitment
-    const proof = computeAggregateKzgProof(blobs);
-    expect(() =>
-      verifyAggregateKzgProof(blobs, commitments, proof),
-    ).toThrowError("verify_aggregate_kzg_proof failed with error code: 1");
-  });
+    it("incorrect commitment", () => {
+      let blob = generateRandomBlob();
+      let commitment = blobToKzgCommitment(generateRandomBlob());
+      let proof = computeBlobKzgProof(blob);
+      expect(verifyBlobKzgProof(blob, commitment, proof)).toBe(false);
+    });
 
-  it("throws the expected error when given fewer commitments than blobs", () => {
-    let blobs = new Array(1).fill(0).map(generateRandomBlob);
-    let commitments = [] as Uint8Array[];
-    let proof = computeAggregateKzgProof(blobs);
-    expect(() =>
-      verifyAggregateKzgProof(blobs, commitments, proof),
-    ).toThrowError(
-      "verifyAggregateKzgProof requires blobs count to match expectedKzgCommitments count",
-    );
+    it("incorrect proof", () => {
+      let blob = generateRandomBlob();
+      let commitment = blobToKzgCommitment(blob);
+      let proof = computeBlobKzgProof(generateRandomBlob());
+      expect(verifyBlobKzgProof(blob, commitment, proof)).toBe(false);
+    });
   });
 
-  describe("computing commitment from blobs", () => {
-    it("throws as expected when given an argument of invalid type", () => {
-      // @ts-expect-error
-      expect(() => blobToKzgCommitment("wrong type")).toThrowError(
-        "Expected blob to be UInt8Array of 131072 bytes",
-      );
+  describe("verifyBlobKzgProofBatch", () => {
+    it("zero blobs/commitments/proofs", () => {
```

Comment:
These tests previously read as sentences describing the expected behavior. They should remain doing so.

Reply:
I understand. Will fix.